variables.tf

```
@@ -48,22 +63,27 @@ variable "region" {
   description = "The region this bucket should reside in, defaults to the region used by the callee"
 }
 
-variable "cors_rule" {
+variable "replication_configuration" {
   type = object({
-    allowed_headers = list(string)
-    allowed_methods = list(string)
-    allowed_origins = list(string)
-    expose_headers  = list(string)
-    max_age_seconds = number
+    iam_role_arn       = string
+    dest_bucket        = string
+    dest_storage_class = string
+    rule_id            = string
   })
   default     = null
-  description = "The CORS rule for the s3 bucket"
+  description = "Object containing cross-region replication configuration."
```

```suggestion
  description = "Object containing bucket replication configuration."
```

This should be updated because it's not only for cross-region replication, we're doing it in the same region right?

---

done